### ISSUE TITLE
Atomic Disassembler destroys instead of mining anything that requires a pickaxe. 

### ISSUE THREAD

**Issue by EagleHamlet** [Divine-Journey-2/main#316]
**Modpack version:**
2.3.1.

**Description:**
Made an Atomic Disassembler, tried to break an empowerer from Actually Additions, it deleted the item. I tried to break wood & dirt and it gave me the drops from those blocks. I tried to break stone afterwards, it deleted the stone. Then I made a furnace, I broke it, it destroyed the drop.

**Steps to reproduce:**
1. Make the item
2. Try to mine anything that requires a pickaxe
3. It destroys the drop.

**Expected behavior:**
It should act as a normal pickaxe and mine the stone, not destroy it. 

Edit: Is it really efficiency 20? Cause it doesn't break stone instantly. A speed increase should also be added. I'm using fastest option and it's quite slow... It breaks dirt instantly at least.

**Comment by Atricos** [comment]
That's apparently just how to Atomic Disassembler works. There are no configs to change the block destroying behaviour. (It makes sence looking at its name.)

There are 5 modes for it, you'll need to toggle between these for a faster mining speed: https://wiki.aidancbrady.com/wiki/Atomic_Disassembler

**Comment by Ringedtech203** [comment]
It shouldn't destroy the blocks

**Comment by Atricos** [comment]
Well, there are no configs for that.

**Comment by Atricos** [comment]
Actually, all I can think of is changing this in `config/tool_progression/tool_overwrites.cfg`:
```
S:"mekanism:atomicdisassembler"=pickaxe=4
S:"mekanism:atomicdisassembler"=shovel=4
S:"mekanism:atomicdisassembler"=axe=4
```
to
```
S:"mekanism:atomicdisassembler"=pickaxe=4,shovel=4,axe=4
```

**Comment by Ringedtech203** [comment]
changing to
`s:"mekanism:atomicdisassembler"=pickaxe=4,shovel=4,axe=4` 
fixes the issue

**Comment by Ringedtech203** [comment]
Also breaks dirt instantly on normal(20) and stone instantly in fast mode(128)

**Comment by Atricos** [comment]
Implemented the change in 2.4.0 (https://www.curseforge.com/minecraft/modpacks/divine-journey-2/files/3464189), which should fix this issue.
